src/main/java/net/mcreator/workspace/Workspace.java

```diff
@@ -294,6 +294,29 @@ public long getMCreatorVersion() {
 		fileManager.close();
 	}
 
+	@Override public boolean equals(Object o) {

```

Comment:
Could this be another PR? This may be a good idea to have implemented outside just this PR

Reply:
#4946